### ISSUE TITLE
No Glyphs on *.razor files

### ISSUE THREAD

**Issue by pauldavis-dwe** [FortuneN/FineCodeCoverage#421]
### Installed product versions
- Visual Studio: 2022 Professional
- This extension: 1.1.214

### Description
After successfully getting coverage from running tests, glyphs do not show on .razor files (but do on .cs files)

### Steps to recreate
1. Run tests
2. See no glyphs on .razor files

### Current behavior
No glyphs on .razor files

### Expected behavior
Glyphs on .razor files

### Side Notes
- It helps to attach a repro solution so i can test the fix : https://github.com/pauldavis-dwe/FccBugRepro2.git
- It helps to attach the log: https://pastebin.com/7w4gUvbZ

**Comment by tonyhallett** [comment]
Thanks Paul for reporting and for attaching the repo.  I will look at this.  In the meantime you can put the C# code in a [code-behind file ](https://learn.microsoft.com/en-us/aspnet/core/blazor/components/?view=aspnetcore-8.0#partial-class-support)

**Comment by pauldavis-dwe** [comment]
Thanks, I do on my production projects

**Comment by tonyhallett** [comment]
Available in latest release.

**Comment by pauldavis-dwe** [comment]
Erm... Glyphs in .cs files now don't work...

**Comment by tonyhallett** [comment]
Thanks Paul will check

**Comment by tonyhallett** [comment]
Glyphs should work although I have changed the functionality to be strict.

If I file has been changed outside of the document then the coverage marks can no longer be expected to apply.
If you have the file open and change it after test execution starts then again the marks can no loner be expected to apply.

If you check the FCC output window pane there should be a message when this occurs.

I can add a boolean option to override this behaviour.
